src/resharper/imports/evaluator-manifest.yaml

```diff
@@ -0,0 +1,71 @@
+queries:
+  issueCountByLevel:
+    query: SELECT COUNT(1) FROM Issues WHERE Level = $Level
+requests:
+  totalFilesCount:
+    query: SELECT COUNT(1) AS count FROM FileDetails
+    type: Scalar
+  totalCodeCount:
+    query: SELECT SUM(Code) FROM FileDetails WHERE Language IN ('C#', 'cshtml')
+    type: Scalar
+  totalLanguagesFilesCount:
+    query: SELECT COUNT(1) as count FROM FileDetails WHERE Language IN ('C#', 'cshtml')
+    ranges:
+      <b><span style='color:red'>(TODO) Number of issues is considered to be high</span></b>:
+        start: 0
+  totalErrorCount:
+    queryRef: issueCountByLevel
+    type: Scalar
+    parameters:
+      Level: Error
+  totalInfoCount:
+    queryRef: issueCountByLevel
+    type: Scalar
+    parameters:
+      Level: Info
+  totalWarningCount:
+    queryRef: issueCountByLevel
+    type: Scalar
+    parameters:
+      Level: Warning
+  topFiveLanguages:
+    query: SELECT Language, count(1) AS count FROM FileDetails GROUP BY Language ORDER BY count DESC LIMIT 5
+    type: Collection
+  topTenBiggestFilesForLanguages:
+    query: SELECT FilePath, Code, CASE WHEN Code < 100 THEN 'S' WHEN Code > 100 AND Code < 300 THEN 'M' WHEN Code > 300 AND Code < 500 THEN 'L' ELSE 'XL' END Size FROM FileDetails WHERE Language IN ('C#', 'cshtml') ORDER BY Code DESC LIMIT 10
+    type: Collection
+  topTenFrequentIssues:
+    query: Select Count(1) as Count, Level, Title, Description from Issues GROUP BY Level, Description, Category ORDER BY Count DESC LIMIT 10
+    type: Collection
+  topTenSevereIssues:
+    query: Select Count(1) as Count, Level, Title, Description from Issues GROUP BY Level, Description, Category ORDER BY Count DESC LIMIT 10
+    type: Collection
+  topTenIssuesByDescription:
+    query: SELECT Description, Count(1) AS count FROM Issues GROUP BY Description ORDER BY Count DESC LIMIT 10
+    type: Collection
+  biggestFileSize:
+    query: SELECT Code FROM FileDetails WHERE Language IN ('C#', 'cshtml') ORDER BY Code DESC LIMIT 1
+    ranges:
+      <b><span style='color:red'>(TODO) File size is considered to be unmaintainable</span></b>:
```

Comment:
Does it make sense to move this HTML into Liquid templtate?

Reply:
To highlight a bad result. Different colors are set for bad and good results.